tests/test_PH.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import uuid
+from pathlib import Path
+
+import pytest_asyncio
+
+from app.core.groups.groups_type import GroupType
+from app.modules.ph import models_ph
+from tests.commons import (
+    add_object_to_db,
+    client,
+    create_api_access_token,
+    create_user_with_groups,
+    event_loop,  # noqa
+)
+
+
```

Comment:
I think we need to define global variables here, but I'm not sure

```suggestion
ph_user_ph: models_core.CoreUser
ph_user_simple: models_core.CoreUser
token_simple: str
```